airflow/providers/jenkins/operators/jenkins_job_trigger.py

```diff
@@ -71,7 +71,6 @@ def jenkins_request_with_headers(jenkins_server: Jenkins, req: Request) -> Jenki
         raise jenkins.TimeoutException(f"Error in request: {e}")
     except URLError as e:
         raise JenkinsException(f"Error in request: {e.reason}")
-    return None
```

Comment:
Do we need to update/correct the return type annotation?

Reply:
Yeah, it never be None